apps/system/js/edge_swipe_detector.js

```diff
@@ -147,9 +147,7 @@
           this.lifecycleEnabled = false;
           break;
         case 'cardviewclosed':
-          if (e.detail && e.detail.newStackPosition) {
-            this.lifecycleEnabled = true;
-          }
+          this.lifecycleEnabled = true;
```

Comment:
Exiting the cardview to the homescreen will enable the edge zones with this change. We don't want that :)
But it should be fine with just checking that the next app isn't a homescreen.